content/design-systems-for-developers/react/en/build.md

```diff
@@ -138,92 +152,48 @@ export const buttonWrapper = () => (
 )
```

Comment:
Can we fix this formatting?

Reply:
Also I don't think we need `action` any more here if the stories use args.

Reply:
Formatting is fixed. Going to take a look at the action and if not necessary i'll update the snippet and commit it.

Reply:
Just committed it. Forgot about that one. Thanks for it.